### DISCUSSION TITLE
Releasing images of a FUZIX port

### DISCUSSION THREAD

**Discussion by ajcasado** [EtchedPixels/FUZIX#1121] [General]
Hi @Etchedpixels,

I’ve been asked here: https://www.cpcwiki.eu/forum/index.php?msg=245142 to make the builds I’ve been generating of FUZIX for the CPC6128 available as GitHub releases. For now, I’m sharing a Dropbox link for this purpose.

I wanted to check with you because I’m not sure what policy you have regarding this. Personally, I think it’s a good idea for the community to have easy access to the latest build so they can experiment with it and provide bug reports and suggestions for improvements. What’s your take on this?

**Comment by EtchedPixels** [comment]
Feel free.

**Reply by ajcasado** [reply]
Thanks for your reply!
